test/unit/ajax.js

```diff
@@ -1410,6 +1410,22 @@ QUnit.module( "ajax", {
 		};
 	} );
 
+	ajaxTest( "jQuery.ajax() - don't escape %20 in contentType (gh-4119)", 1, function( assert ) {
+		return {
+			url: "bogus.html",
+			contentType: "application/x-www-form-urlencoded",
+			headers: { "content-type": "application/json" },
```

Comment:
We should also have a test with these values reversed to make sure the override still happens.

Reply:
Done; added a new test.